### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by katjuell**
Resolves [this Asana ticket](https://app.asana.com/0/1200198213175654/1200632438869796/f) by adding a PR template that will outline both acceptance criteria and guidelines for testing.

Open to suggestions on wording etc.!

### CODE REVIEW

**Comment by marekweb** [comment]
I tried a variation of the template when I wrote #121 to get a feel for the sections.

Some thoughts:

- For "type of change", what do you think about putting that in GitHub PR labels? I find the labels really helpful when searching and filtering in the PR list.
- One of the first questions that come to my mind when I look at a new PR is, "why are we making this change? / what's the bigger picture or rationale?" I find a "background context" or "why" section to be useful - would that make sense as a separate section?
- "Leave a comment on things you want explicit feedback on" - I like this, it's great advice!
- Since we're in the process of creating documentation for our team's processes and conventions: once we have a link to that document (in the repo or in the company handbook, wherever it will end up), we could move the "Pull request process" section there, too (and link to it). What do you think?

**Comment by ltagliaferri** [comment]
I agree that we can use labels vs. type of change, that can make the process more lightweight for the person opening a PR, and can make it easier for maintainers (etc.) to sort through tickets later. 

@marekweb — regarding context / rationale, do you think that that would be in lieu of or in addition to a Jira / Asana ticket? It would be great if we could balance enough context to make our work really clear without overloading the person opening the PR with duplicative work too.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+## Type of change
+<!-- Please be sure to add the appropriate label to your PR. -->
```
Comment (katjuell) [inline .github/PULL_REQUEST_TEMPLATE.md:2]: Should we invite folks to add labels if they don't see one that fits?

**Comment by katjuell** [comment]
Thanks for the feedback! Hopefully these changes capture everything. I posted my one question above.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (ltagliaferri) [reply .github/PULL_REQUEST_TEMPLATE.md:2]: Yes, I just added one 😄

**Review by ltagliaferri — APPROVED** [review]
This is great, thank you

**Pull request merged**